packages/firestore/src/local/simple_db.ts

```diff
@@ -342,6 +360,14 @@ export class SimpleDbStore<KeyType extends IDBValidKey, ValueType> {
     return wrapRequest<void>(request);
   }
 
+  // if we ever need more of the count variants, we can add overloads. For now,
```

Comment:
Suggestion: /** Counts all objects in the store. Runs in O(n) */

In my opinion, The part about the overloads doesn't seem to warrant a comment.

Reply:
I left the part about overloads because we are explicitly not supporting part of the underlying API, whereas other methods in this class provide overloads to handle all of the variants of the API calls that they are wrapping. As this is a deviation from what we are currently do, I thought it warranted a comment.

I added the `O(n)` note though.